static/app/components/events/featureFlags/featureFlagOnboardingLayout.tsx

```diff
@@ -95,28 +93,32 @@ export function FeatureFlagOnboardingLayout({
   return (
     <AuthTokenGeneratorProvider projectSlug={projectSlug}>
       <Wrapper>
-        {!skipConfig ? null : (
+        {!skipEvalTracking ? null : (
           <Alert.Container>
             <Alert type="info" showIcon>
               <Flex gap={space(3)}>
                 {t(
                   'Feature flag integration detected. Please follow the remaining steps.'
                 )}
-                <Button onClick={() => setSkipSteps(!skipSteps)}>
-                  {skipSteps ? t('Show Full Guide') : t('Hide Full Guide')}
+                <Button onClick={() => setHideSteps(!hideSteps)}>
+                  {hideSteps ? t('Show Full Guide') : t('Hide Full Guide')}
                 </Button>
               </Flex>
             </Alert>
           </Alert.Container>
         )}
-        {!skipSteps && (
+        {!hideSteps && (
```

Comment:
can we phrase the condition in the positive here? `{hideSteps ? null : <Steps>...</Steps>}`